tests/test_commands.py

```diff
@@ -415,3 +415,81 @@ def mockreturn_two_shards(self, conn):
     run_set_replicas(args)
 
     assert mock_log.called
+
+
+@patch('argparse.ArgumentParser.parse_args')
+@patch('bigchaindb.commands.utils.base_parser')
+@patch('bigchaindb.commands.utils.start')
+def test_calling_main(start_mock, base_parser_mock, parse_args_mock,
```

Comment:
I think @vrde proposed a better way to test this. Maybe he can comment on this and tell us what to do.